universum/modules/structure_handler.py

```diff
@@ -78,11 +78,10 @@ def is_successful(self) -> bool:
 class StructureHandler(Module):
     def __init__(self, *args, **kwargs):
         super(StructureHandler, self).__init__(*args, **kwargs)
-        block_structure = Block("Universum")
-        self.current_block = block_structure
-        self.configs_current_number = 0
-        self.configs_total_count = 0
-        self.active_background_steps = []
+        self.current_block: Block = Block("Universum")
+        self.configs_current_number: int = 0
+        self.configs_total_count: int = 0
+        self.active_background_steps: List[Callable[[Any], Any]] = []
```

Comment:
we should really apply some restrictions here, but I don't know much about possible values in 'active_background_steps'

Reply:
It requires explicitly defining class for background step.
This is a simple refactoring and you can do it as a part of this PR.

Reply:
Callable and Any are not defined